lib/ReactTags.js

```diff
@@ -7,7 +7,7 @@ import PropTypes from "prop-types";
 import Tag from "./Tag";
 
 //Constants
-import { KEYS, DEFAULT_PLACEHOLDER, DEFAULT_CLASSNAMES } from './constants';
+import { KEYS, DEFAULT_PLACEHOLDER, DEFAULT_CLASSNAMES } from "./constants";
```

Comment:
single quotes ?

Reply:
This was the output of npm run format.

Reply:
@cklab i just merged a commit which has changed the configuration of prettier so please run it again.

Reply:
I see, done.